src/plugin.js

```diff
@@ -1,206 +1,202 @@
-/* global window: false */
+/* global window: false, ChartJS */
 'use strict';
 
-(function() {
-
-	var Chart = window.Chart;
-	var helpers = Chart.helpers;
-	var STUB_KEY = '_chartjs_deferred';
-	var MODEL_KEY = '_deferred_model';
-
-	/**
-	 * Plugin based on discussion from Chart.js issue #2745.
-	 * @see https://github.com/chartjs/Chart.js/issues/2745
-	 */
-	Chart.Deferred = Chart.Deferred || {};
-	Chart.Deferred.defaults = {
-		enabled: true,
-		xOffset: 0,
-		yOffset: 0,
-		delay: 0
-	};
-
-	// DOM implementation
-	// @TODO move it in Chart.js: src/core/core.platform.js
-	Chart.platform = helpers.extend(Chart.platform || {}, {
-		defer: function(fn, delay, scope) {
-			var callback = function() {
-				fn.call(scope);
-			};
-			if (!delay) {
-				helpers.requestAnimFrame.call(window, callback);
-			} else {
-				window.setTimeout(callback, delay);
-			}
-		}
-	});
-
-	function computeOffset(value, base) {
-		var number = parseInt(value, 10);
-		if (isNaN(number)) {
-			return 0;
-		} else if (typeof value === 'string' && value.indexOf('%') !== -1) {
-			return number / 100 * base;
+var Chart = ChartJS || window.Chart;
```

Comment:
Changing the whole file indentation by removing the surrounding function make the PR almost unreadable and will inevitably conflict with the upcoming changes I'm working on based on Chart.js 2.6. Also, I would like to keep that file directly injectable (`<script>`) and thus preserve the global function scoping. That's why I prefer the following approach (inject the UMD param as function parameter):

```javascript
/* global window: false, Chart */

'use strict';

(function(Chart) {

    // remove: var Chart = window.Chart;
    // and keep previous code unchanged
    // ...

})(Chart || window.Chart);
```